Resizetizer.NT/Resizetizer.NT.targets

```diff
@@ -54,6 +68,12 @@
 			$(ResizetizeAfterTargets);
 			ResizetizeCollectItems;
 		</ResizetizeAfterTargets>
+
+		<ProcessSharedFontsAfterTargets>
+			$(ProcessSharedFontsAfterTargets);
+			_CompileCoreMLModels;
```

Comment:
😬  Is this the best place?

Reply:
For now, eventually there will be a better hook as per https://github.com/xamarin/xamarin-macios/issues/10646